numba_dpex/tests/experimental/test_inputs.py

```diff
@@ -0,0 +1,33 @@
+import dpctl.tensor as dpt
```

Comment:
The test case should be renamed to `test_supported_array_types_as_kernel_args.py`. Verbose but captures what the test is checking.